apps/comps/app/competitions/[id]/chart/page.tsx

```diff
@@ -37,10 +36,12 @@ export default function CompetitionChartPage({
     data: agentsData,
     isLoading: agentsLoading,
     error: agentsError,
-  } = useCompetitionAgents(id, {
-    limit: 10, // Default page size
-    offset: 0,
-  });
+  } = useQuery(
+    tanstackClient.competitions.getAgents.queryOptions({
```

Comment:
`useCompetitionAgents` previously had an "enabled" check based on if the competitionId was defined or not.  Not all the places where you inlined the query carried over that "enabled" arg.  I think there are a few others throughout this PR besides just this one

Reply:
In this particular case, `id` is always defined.